module3/assignment3.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Keating | INFO 2439 | Assignment #3</title>
+  <meta charset="utf-8">
+  
+  <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css" />
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+  <script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+  <link href="https://fonts.googleapis.com/css?family=Reenie+Beanie&display=swap" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" href="_css/keating_a3.css">
+</head>
+<body>
+<!-- Collin Keating
+     assignment3.html
+     INFO2439
+     Thoendel
+     9/29/19
+-->
+<div data-role="page" id="homePage" data-title="Keating | INFO 2439 | Assignment #3" data-add-back-btn="true" data-back-btn-text="Back">
+<div data-role="header" data-position="fixed"><a data-transition="slide" href="#" data-direction="reverse" class="ui-btn-left ui-btn ui-btn-inline ui-mini ui-corner-all ui-btn-icon-right ui-icon-back">Back</a></div>
+<div role="main" class="ui-content">
+            <h2>Mobile Application Research</h2>
+            <ul data-role="listview" data-inset="true">
+                <li><a data-transition="slide" href="ctheme1/index.html">First</a></li>
+                <li><a data-transition="slide" href="ctheme2/index.html">Second</a></li>
```

Comment:
The reason these links don't seem to show the css theme until refresh is because jQuery mobile use ajax to load these pages and *skips* the head portion to save load times. To alter this behavior add the attribute data-ajax="false" to the anchor tags. That will force the browser to reload all head links - and your new css will apply. let me know if you want help with this.